exist-core/src/main/java/org/exist/collections/triggers/DocumentTriggers.java

```diff
@@ -53,7 +53,9 @@
     private SAXTrigger last = null;
     
     private final List<DocumentTrigger> triggers;
-    
+
+    private boolean skipTriggers = false;
```

Comment:
Likewise... These `triggersSuspended` checks should not be used here. Such checks are already present at the call site in DBBorker.